src/desktop/analytics/articles.js

```diff
@@ -72,28 +45,6 @@ if (location.pathname.match('/article/') || location.pathname.match('/2016-year-
   })
 
   // Hooks
-  analyticsHooks.on('readmore', function (options) {
-    analytics.track('Clicked Read More', {})
-    analytics.page({path: location.pathname}, {integrations: {'Marketo': false}})
-    if (window.PARSELY) {
-      window.PARSELY.beacon.trackPageView({
-        url: location.href,
-        urlref: sd.APP_URL + '/' + options.urlref,
-        js: 1,
-        action_name: 'infinite'
-      })
-    }
-    if (window.Sailthru) {
-      window.Sailthru.track({
-        domain: 'horizon.artsy.net',
-        spider: true,
-        track_url: true,
-        url: sd.APP_URL + '/' + location.pathname,
-        use_stored_tags: true
-      })
-    }
-  })
-
```

Comment:
Removed a bunch of track calls we no longer use since we depend on Reaction now.